modules/private-cluster/main.tf

```diff
@@ -157,6 +157,8 @@ data "google_container_engine_versions" "region" {
 }
 
 data "google_container_engine_versions" "zone" {
+  provider = "google-beta"
```

Comment:
This change is not relevant to the branch topic.

Reply:
Same as above.